### DISCUSSION TITLE
Widget script no longer loads

### DISCUSSION THREAD

**Discussion by alexsol22** [Laravel-Backpack/community-forum#1182] [Q&A (Help)]
I'm having problems loading a script widget which is strange because it worked until recently. 
I tried two different ways but nothing:
1. `Widget::add()->type('script')->content('assets/js/admin/forms/contract.js');`
2. `Widget::add()->type('script')->content(asset('assets/js/admin/forms/contract.js'));`

what could be wrong?

**Comment by karandatwani92** [comment]
Hey @alexsol22 

I checked the demo to confirm. It's working fine. It could be something else at your end.

Here are a few links for reference:
- Demo: https://demo.backpackforlaravel.com/admin/field-monster/138/edit
- Setup: https://github.com/Laravel-Backpack/demo/blob/1aad77850c23fc125068442fec225f7f635c25a8/app/Http/Controllers/Admin/FieldMonsterCrudController.php#L26
- JS Assets: https://github.com/Laravel-Backpack/demo/tree/main/public/assets/js/monster
